packages/app/hooks/use-block.ts

```diff
@@ -57,9 +57,9 @@ function useBlock() {
         }),
         true
       );
-      toast?.show({ message: "Blocked!", hideAfter: 4000 });
+      toast("Blocked!");
```

Comment:
is this API valid? `toast(message)`

Reply:
yeah, if this method will be without any icon. 
I just think that the `blocked` and `reported` is a little bit negative, so I didn't add any positive icons to the above. 😃